tests/riscv64/smoke.sh

```diff
@@ -50,7 +50,7 @@ function generate_llvm_cpu_vmfb {
       --iree-input-type=tosa
       --iree-llvm-target-cpu-features="+m,+a,+f,+d,+c,+v"
       --riscv-v-fixed-length-vector-lmul-max=8
-      --riscv-v-vector-bits-min=256
+      --riscv-v-vector-bits-min=512
```

Comment:
Do you also want to update https://github.com/iree-org/iree/blob/main/docs/website/docs/building-from-source/riscv.md#optional-configuration / https://iree-org.github.io/iree/building-from-source/riscv/ ?

Reply:
Nice catch! Will do.